modules/rosa-hcp-create-network.adoc

```diff
@@ -89,4 +364,75 @@ INFO[0140] Resource: VPC, Type: AWS::EC2::VPC, ID: <vpc-id>
 INFO[0140] Stack rosa-network-stack-5555 created \ <2>
 ----
 <1> These two subnet IDs are used to create your cluster when using the `rosa create cluster` command.
-<2> The network stack name is used to delete the resource later.
\ No newline at end of file
+<2> The network stack name is used to delete the resource later.
+
+ifdef::rosa-egress-lockdown[]
+[discrete]
+[id="rosa-hcp-vpc-subnet-tagging-rosa-network_{context}"]
+== Tagging your subnets
+
+Before you can use your VPC to create a {hcp-title} cluster, you must tag your VPC subnets. Automated service preflight checks verify that these resources are tagged correctly. The following table shows how your resources should be tagged:
```

Comment:
```suggestion
Before you can use your VPC to create a {hcp-title} cluster, you must tag your VPC subnets. Automated service preflight checks verify that these resources are tagged correctly. The following table shows how to tag your resources:
```

- Active voice + should is avoided

https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s

> Whenever possible, rewrite in a more direct way. Do not use should to refer to an action that must be performed.